DEVELOPER_SETUP.md

```diff
@@ -48,6 +48,13 @@ Then create the virtual environment
 poetry install --sync
 ```
 
+If you are on Archlinux you need to run the command as follows:
+
+```sh
+# https://bbs.archlinux.org/viewtopic.php?id=296542
```

Comment:
The info link is duplicated

Reply:
We could also mention the open PR that, when merged, will allow us to remove this specific instruction:
https://github.com/mariadb-corporation/mariadb-connector-python/pull/29

Reply:
Totally agree!

Reply:
oh. My bad
I forgot to remove it from behind the instruction :laughing: 
Will remove real quick